NAMESPACE

```diff
@@ -17,7 +17,6 @@ export(CH3BR_CONSTRAIN)
 export(CH3CCL3_CONSTRAIN)
 export(CH3CL_CONSTRAIN)
 export(CH4_CONSTRAIN)
-export(CO2_CONSTRAIN)
```

Comment:
I flagged this as an issue with #639 we are losing a lot of R wrapper functions with this commit.

Reply:
I don't understand what's happening here.